taichi/backends/cuda/cuda_device.cpp

```diff
@@ -99,6 +99,19 @@ uint64 CudaDevice::fetch_result_uint64(int i, uint64 *result_buffer) {
                                                    sizeof(uint64));
   return ret;
 }
+
+CudaCommandList::CudaCommandList(CudaDevice *ti_device)
```

Comment:
Command list is supposed to only record commands and defer the execution until the submit call. So this isn't the correct behavior here